src/galileo/logger/logger.py

```diff
@@ -134,35 +135,52 @@ def __init__(
         project_name_from_env = getenv("GALILEO_PROJECT", DEFAULT_PROJECT_NAME)
         log_stream_name_from_env = getenv("GALILEO_LOG_STREAM", DEFAULT_LOG_STREAM_NAME)
 
+        project_id_from_env = getenv("GALILEO_PROJECT_ID")
+        log_stream_id_from_env = getenv("GALILEO_LOG_STREAM_ID")
+
         self.project_name = project or project_name_from_env
+        self.project_id = project_id or project_id_from_env
```

Comment:
Unrelated, but this is exactly where using `galileo_core.BaseConfig` would simplify a ton of this logic. We allow init from env vars (type-cast with Pydantic) or as args, and it all just works...